src/displayBackend/wayland/Connector.cpp

```diff
@@ -80,6 +80,15 @@ void Connector::pageFlip(FrameBufferPtr frameBuffer, FlipCallback cbk)
 	dynamic_cast<WlBuffer*>(frameBuffer.get())->setSurface(mSurface);
 }
 
+void Connector::stopProcessing()
+{
+	if (mSurface)
```

Comment:
The same here. Only this connecter is owning the surface. If so, disable callback can be called in surface destructor.

Reply:
Done